k6ReportPortal.js

```diff
@@ -37,92 +42,132 @@ export function finishLaunch(launchId,reporterOptions) {
  * @returns 
  */
 function getHeader(token) {
-    var header = {
+    return {
         headers: {
-            "Content-Type": "application/json",
+            'Content-Type': 'application/json',
             authorization: `Bearer ${token}`,
         },
     };
-    return header;
-}
-
+};
 
 /**
  * 
  */
 export default class RpClient {
-
-    constructor(launchId,reporterOptions){
+    constructor(launchId, reporterOptions) {
         this.launchId = launchId
         this.reportPortalUri = `${reporterOptions.endpoint}/${reporterOptions.project}`
         this.token = reporterOptions.token
+    }
 
+    addLogToBatch(jsonObject, message, level = 'error') {
```

Comment:
Can you please add documentation string and explain what this method does.
/**

*/